impls/msft-green-ai.yaml

```diff
@@ -0,0 +1,44 @@
+name: mft-green-ai
+description: "https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-green-ai.md"
+tags:
+  kind: ml
+  complexity: simple
+  category: cloud
+config:
+  funit:
+    - name: job
+      scale-factor: 1 # divide SCI by this value to get SCI per ml job
+  models:
+    - name: sci-c # a model that given e, i and m calculates a carbon value (e * i) + m
+      path: builtin       
+graph:
+  DenseNet: # an advanced grouping node
+    pipeline: 
+      - sci-c # energy & grid-carbon-intensity & embodied -> carbon
+    config: 
+      sci-c:
+        processor: nvidia-t4-nc16as-v3
+        e: 17.97
```

Comment:
This is an observation I would say, the core one (GPUs are the only things that actually report energy directly)
Add to the observation as "energy"